content/en/blog/2022/demo-announcement.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Announcing a Community Demo for OpenTelemetry
+linkTitle: OpenTelemetry Community Demo
+date: 2022-06-16
+author: Carter Socha
+---
+
+As OpenTelemetry matures, users are increasingly looking for best practice
+guides on how to onboard their services to the new paradigm or demo applications
+so that they can try out the new tools themselves.
+
+Multiple vendors have written their own demo applications but are wholly
+responsible for the development & ongoing support. The existing demos are all
+feature incomplete in their own ways with missing languages, restrictions on
+backend choice, & they’re overly reliant on instrumentation libraries.
+
+The OpenTelemetry Community Demo application is intended to be a ‘showcase’ for
+OpenTelemetry API, SDK, and tools in a “production-ish” cloud native
+application. The overall goal of this application is not only to provide a
+canonical ‘demo’ of OpenTelemetry components, but also to act as a framework for
+further customization by end-users, vendors, and other stakeholders.
+
+## Project Goals
+
+* Provide developers with a robust sample application they can use
+in learning OpenTelemetry instrumentation.
+* Provide observability vendors with a
+single, well-supported, demo platform that they can further customize or simply
+use OOB.
+* Provide the OpenTelemetry community with a living artifact that
+demonstrates the features and capabilities of OTel APIs, SDKs, and tools.
+* Provide OpenTelemetry maintainers and working groups a platform to demonstrate
+new features/concepts ‘in the wild’
+
+## Current State
+
+As a starting point, we have selected a fork of the popular GCP
+microservices demo. Our first feature additions have been to simplify local
+deployment by consolidating the project onto a single docker compose file,
+updating the documentation, & replacing a pre-existing service with a Ruby
+example. Otherwise the pre-existing feature set from the GCP demo remains the
+same:
+
+* 10 application microservice with support for 6 languages (C#, Go, Java, Node.js,
+Python, & Ruby)
+  * Ruby support was added within the last 2 weeks of publishing
+date
+* Designed to work on docker locally
+* Uses redis cache
+* Auto-instrumentation using instrumentation libraries Tracing support for the gRPC, Redis, & HTTP
+libraries
+* Jaeger visualizations for distributed traces OpenTelemetry collector
+* Always on sampling (100% of telemetry is submitted) Synthetic load generation
+
+## Current Architecture
+
+!["Screenshot of the current architecture."](../../../../static\img\blog\current-demo-architecture.png/)
+
+/current-demo-architecture.png "Screenshot of the current architecture")
```

Comment:
Delete this and add the alt text to ln57

Reply:
Okay I think everything is updated. Barring any other reviews it looks good from my end & ready to be scheduled to be published on Monday, the 20th. 

Semi unrelated but I'd be interested in any user stats after we release it